jujugui/static/gui/src/app/components/entity-details/content/content.js

```diff
@@ -555,9 +555,10 @@ class EntityContent extends React.Component {
     const entity = entityModel.toEntity();
     const storeId = entity.type === 'charm' ?
       entity.storeId : entity.id.split('cs:').join('');
+    const dataDD = this.props.flags.ddeploy ? 'data-dd' : '';
```

Comment:
this is what tells the cards to use the dd link. If you would like that changed a new version of the cards library will need to be created.